Formula/librasterlite2.rb

```diff
@@ -0,0 +1,102 @@
+class Librasterlite2 < Formula
+  desc "Library to store and retrieve huge raster coverages"
+  homepage "https://www.gaia-gis.it/fossil/librasterlite2/index"
+  url "https://www.gaia-gis.it/gaia-sins/librasterlite2-sources/librasterlite2-1.1.0-beta1.tar.gz"
```

Comment:
you need to update the `audit_exceptions/unstable_allowlist.json` to get the build going.